modules/gltf/register_types.cpp

```diff
@@ -30,23 +30,9 @@
 
 #include "register_types.h"
 
-#ifndef _3D_DISABLED
```

Comment:
Why was this condition removed?

Reply:
It did nothing before, the GLTF module already has checks in SCons to not compile when 3D is disabled.